Subject: fix test for 7.3 (int vs integer)
Author: Remi Collet <remi@remirepo.net>
Origin: https://git.remirepo.net/cgit/rpms/php/php-geos.git/tree/0001-fix-test-for-7.3-int-vs-integer.patch
Bug: https://git.osgeo.org/gitea/geos/php-geos/issues/25

--- a/tests/004_WKBWriter.phpt
+++ b/tests/004_WKBWriter.phpt
@@ -139,7 +139,7 @@ class WKBWriterTest extends GEOSTest
             $writer->write(1);
             $this->assertTrue(false);
         } catch (ErrorException $e) {
-            $this->assertContains('expects parameter 1 to be object, integer given', $e->getMessage());
+            $this->assertContains('expects parameter 1 to be object, int', $e->getMessage());
         }
     }
 
@@ -151,7 +151,7 @@ class WKBWriterTest extends GEOSTest
             $writer->writeHEX(1);
             $this->assertTrue(false);
         } catch (ErrorException $e) {
-            $this->assertContains('expects parameter 1 to be object, integer given', $e->getMessage());
+            $this->assertContains('expects parameter 1 to be object, int', $e->getMessage());
         }
     }
 }
