Description: fix __toString with 8.2
Author: Remi Collet <remi@remirepo.net>
Origin: https://git.remirepo.net/cgit/rpms/php/php-geos.git/plain/0006-fix-__toString-with-8.2.patch
Bug: https://git.osgeo.org/gitea/geos/php-geos/issues/26

--- a/arginfo.h
+++ b/arginfo.h
@@ -36,7 +36,12 @@ ZEND_END_ARG_INFO()
 
 #define arginfo_Geometry_construct       arginfo_void
 
+#if PHP_VERSION_ID >= 80200
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_Geometry_toString, 0, 0, IS_STRING, 0)
+ZEND_END_ARG_INFO();
+#else
 #define arginfo_Geometry_toString        arginfo_void
+#endif
 
 ZEND_BEGIN_ARG_INFO_EX(arginfo_Geometry_project, 0, 0, 1)
     ZEND_ARG_INFO(0, geom)
